docs/src/content/docs/api/calls.mdx

```diff
@@ -147,49 +147,59 @@ An optional query parameter called `priorCallId` can be provided to continue a p
         <th class="w-10/12"></th>
       </tr>
       <tr>
-        <td class="font-mono">systemPrompt <br /> <span class="text-sm text-red-500">required</span></td>
+        <td class="font-mono">initator</td>
         <td>string</td>
-        <td>The system prompt to use for the AI.</td>
+        <td>Who was responsible for staring this call. Typically set to agent for outgoing calls and left as the default (user) otherwise.<br />Can be `"agent"` or `"user"` (default).</td>
```

Comment:
```suggestion
        <td>Who was responsible for staring this call. Typically set to INITIATOR_AGENT for outgoing calls and left as the default (INITIATOR_USER) otherwise.</td>
```